Examples/MAX32655/Unity_Test/project.mk

```diff
@@ -0,0 +1,43 @@
+# This file can be used to set build configuration
+# variables.  These variables are defined in a file called 
+# "Makefile" that is located next to this one.
+
+# For instructions on how to use this system, see
+# https://analogdevicesinc.github.io/msdk/USERGUIDE/#build-system
+
+# **********************************************************
+
+# Add your config here!
+LIB_UNITY=1
+
+
+TEST_CC=gcc
+TEST_CFLAGS=-std=c89
+TEST_CFLAGS += -Wall
+TEST_CFLAGS += -Wextra
+TEST_CFLAGS += -Wpointer-arith
+TEST_CFLAGS += -Wcast-align
+TEST_CFLAGS += -Wwrite-strings
+TEST_CFLAGS += -Wswitch-default
+TEST_CFLAGS += -Wunreachable-code
+TEST_CFLAGS += -Winit-self
+TEST_CFLAGS += -Wmissing-field-initializers
+TEST_CFLAGS += -Wno-unknown-pragmas
+TEST_CFLAGS += -Wstrict-prototypes
+TEST_CFLAGS += -Wundef
+TEST_CFLAGS += -Wold-style-definition
+
+
+TARGET_OUT=testbench
+UNITY_DIR=$(LIBS_DIR)/Unity
+TEST_SRC=$(UNITY_DIR)/src/unity.c
+TEST_SRC+=test/test_runner.c
+TEST_SRC+=test/test_functions.c
+
+TEST_SRC+=simple_code.c
+
+TEST_INC += -I$(LIBS_DIR)/Unity/src
+
+unit:
```

Comment:
Might be better to just add to `SRCS`/`IPATH`/`VPATH` instead of defining a custom unit target recipe.

Is the motivation to isolate the unit test from the rest of our compiler flags/build?

Reply:
It doesn't matter if they run the unit test on the MCU, but if it uses the same build flags, it will contain the mmcu flag and other arm specific stuff, so you will get gcc errors when targeting your unit test for x86